selfdrive/modeld/models/driving.cc

```diff
@@ -88,7 +98,7 @@ ModelDataRaw model_eval_frame(ModelState* s, cl_command_queue q,
   net_outputs.long_x = &s->output[LONG_X_IDX];
   net_outputs.long_v = &s->output[LONG_V_IDX];
   net_outputs.long_a = &s->output[LONG_A_IDX];
-  net_outputs.meta = &s->output[META_IDX];
+  net_outputs.meta = &s->output[DESIRE_STATE_IDX];
```

Comment:
This assumes desire state and meta are always next to each other?

Reply:
Yes, at the moment desire_state, other_meta(engaged and disengage prediction), and desire Prediction are grouped together into meta. It's pretty ugly, I wanted to think about refractor before merging